### ISSUE TITLE
How to delete user from one list?

### ISSUE THREAD

**Issue by shrabanee** [igord/sendy-api#5]
Hi,
I want to delete user from sendy's some specific list under some condition. I have list id and user's email. But there is no info about how I can delete the user from code (using javascript). Is there any way to do this?

**Comment by igord** [comment]
By using unsubscribe?

**Comment by shrabanee** [comment]
No, not unsubscribe. One functionality is there to delete user from specific list of sendy, in the view. Like that. Can this be done from code also?
